app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      If maths is your passion, you'll know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.
+    </p>
+
+    <p>
+      An exciting career spent passing your knowledge on to the next generation could be for you.
+    </p>
+
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "quote",
```

Comment:
@caitlin-reidy if we are leaving this block in I think it would be good to add in some examples here of the types of jobs as it's quite broad - or we could keep it broad but give 2 contrasting examples? UR on the subject pages suggests people find it helpful to see the relevance so in turn they can inspire pupils.

Maths is critical to science, technology and engineering as well as being fundamental for most jobs.